src/Compress.jl

```diff
@@ -89,7 +93,13 @@ function save(path::String, uncompressed::Dict,
               versions::Vector{VersionNumber} = load_versions(path))
     compressed = compress(path, uncompressed)
     open(path, write=true) do io
-        TOML.print(io, compressed, sorted=true)
+        if STDLIB_TOML
+            TOML.print(io, compressed, sorted=true) do x
```

Comment:
Can also just pass `string` as the first argument.

Reply:
Yea, I wrote it like this to do some more validation but doesn't really matter.